ios/IJKMediaPlayer/IJKMediaPlayer/ijkmedia/ijksdl/ios/IJKSDLAudioQueueController.m

```diff
@@ -41,6 +41,28 @@ @implementation IJKSDLAudioQueueController {
     NSLock *_lock;
 }
 
+- (void) allocateAudioQueueBuffers {
+    for (int i = 0; i < kIJKAudioQueueNumberBuffers; i++)
+    {
+        if (!_audioQueueBufferRefArray[i]) {
```

Comment:
There is no need to determine the audio queue buffer because it was initialized in constructor of controller